extensions/go-package-search/src/index.tsx

```diff
@@ -0,0 +1,164 @@
+import { ActionPanel, Action, List, showToast, Toast, Icon } from "@raycast/api";
+import { useState, useEffect, useRef, useCallback } from "react";
+import fetch, { AbortError } from "node-fetch";
+import { parse } from "node-html-parser";
+
+export default function Command() {
+  const { state, search } = useSearch();
+
+  return (
+    <List isLoading={state.isLoading} onSearchTextChange={search} searchBarPlaceholder="Search Go packages..." throttle>
+      <List.Section title="Results" subtitle={state.results.length + ""}>
+        {state.results.map((searchResult) => (
+          <SearchListItem key={searchResult.name} searchResult={searchResult} />
+        ))}
+      </List.Section>
+    </List>
+  );
+}
+
+function SearchListItem({ searchResult }: { searchResult: SearchResult }) {
+  return (
+    <List.Item
+      icon={searchResult.search ? Icon.MagnifyingGlass : "list-icon.png"}
+      key={searchResult.fullPath}
+      title={searchResult.name}
+      subtitle={searchResult.description}
+      accessoryTitle={searchResult.fullPath}
+      accessoryIcon={searchResult.standardLibrary ? Icon.Pin : undefined}
```

Comment:
This way is deprecated. Now you need to use [accessories](https://developers.raycast.com/api-reference/user-interface/list#list.item.accessory) instead.

Could be like this:
```suggestion
      accessories={[
        {text: searchResult.fullPath},
        {icon: searchResult.standardLibrary ? Icon.Pin : undefined}

      ]}
```

or like this:
```suggestion
      accessories={[
        {text: searchResult.fullPath, icon: searchResult.standardLibrary ? Icon.Pin : undefined}
      ]}
```

Reply:
Great suggestion! Took the second one in [`555c908` (#1236)](https://github.com/raycast/extensions/pull/1236/commits/555c908bd9ba14dd78e0b8fa37df1da2a8936f93)